### DISCUSSION TITLE
Error reading file mitmproxy-ca-cert.pem

### DISCUSSION THREAD

**Discussion by Ilici-Ramirez** [mitmproxy/mitmproxy#6425] [Q&A]
Hi, i am on macOS Sonoma 14.0
when issuing:
$ sudo security add-trusted-cert -d -p ssl -p basic -k /Library/Keychains/System.keychain mitmproxy-ca-cert.pem

i get:
Error reading file mitmproxy-ca-cert.pem

any suggestions please?

thanks

**Comment by RicardoAnjos** [comment]
After installing, did you run the mitm in terminal?
Please make sure to run Launch the tool you need (mitmproxy or mitmweb) in terminal first so the folder ~/.mitmproxy gets properly created.

That worked for me. Hope it helps

**Comment by vheckthor** [comment]
I `cd` into the directory `~/.mitmproxy`  and run the command `sudo security add-trusted-cert -d -p ssl -p basic -k /Library/Keychains/System.keychain mitmproxy-ca-cert.pem` this is what works for me.
